src/Psy/Command/ParseCommand.php

```diff
@@ -59,12 +68,23 @@ public function setPresenter(Presenter $presenter)
      */
     protected function configure()
     {
+        $definition = array(
+            new InputArgument('code', InputArgument::REQUIRED, 'PHP code to parse.'),
+            new InputOption('depth', '', InputOption::VALUE_REQUIRED, 'Depth to parse', 10),
+        );
+        if ($this->parserFactory->hasKindsSupport()) {
+            $definition[] = new InputOption(
+                'parser-kind',
+                '',
+                InputOption::VALUE_OPTIONAL,
+                "One PhpParser\\ParserFactory constants (default is based on current interpreter's version)",
+                $this->parserFactory->getDefaultKind()
```

Comment:
I'd like to enumerate the possible parser kinds, probably in the `<<<HELP` block below? Also wouldn't hurt to add an example while we're at it :)